xsuite/src/data/index.test.ts

```diff
@@ -842,6 +842,52 @@ test("e.account", async () => {
   );
 });
 
+test("e.MapperKey - simple key", () => {
```

Comment:
Let's put this after all the encoders that start with a uppercase. Also, let's split .toTopHex and .toNestHex in two tests, the same way it is done for other value encodables